docs/docs/dev_docs/contracts/portals/main.md

```diff
@@ -34,20 +34,24 @@ With all that information at hand, we can call the `sendL2Message` function on t
 
 As time passes, a sequencer will see your tx, the juicy fee provided and include it in a rollup block. Upon inclusion, it is removed from L1, and made available to be consumed on L2.
 
-To consume the message, we can use the `consume_l1_to_l2_message` function within the `context` struct. 
+An example usage of this flow is a token bridge. You can deposit to Aztec i.e. mint your tokens on aztec publicly or privately as shown here:
+
+#include_code token_portal_deposit l1-contracts/test/portals/TokenPortal.sol solidity
+
+To consume the message on Aztec, we can use the `consume_l1_to_l2_message` function within the `context` struct. 
```

Comment:
```suggestion
To consume the message on Aztec, we can use the `consume_l1_to_l2_message` function within the `context`. 
```

Not 100% sure saying struct here is required? although i could be being pedantic